### ISSUE TITLE
Browser Synchronisation

### ISSUE THREAD

**Issue by funky123** [linkwarden/linkwarden#245]
**Is your feature request related to a problem? Please describe.**
Export to or Synchronisation with Browsers 

**Describe the solution you'd like**
Getting all Bookmarks to a browser (like nextcloud bookmarks with browser extension) would be killer feature

**Comment by snickers2k** [comment]
yes would be great to have everything in sync

**Comment by zombiehoffa** [comment]
Is there any chance this moves forward? I would love to be able to choose collections/sub collections to sync to firefox/from firefox (well really librewolf, but it should be the same thing here) orchestrated by the plugin possibly.

**Comment by Kasi07** [comment]
As already mentioned above, nextcloud bookmarks or rather floccus (https://github.com/floccusaddon/floccus) is a good starting point as they sync native browser bookmarks to nextcloud/webdav/etc.

**Comment by marcelklehr** [comment]
Creator of floccus here, does linkwarden have a public API?

**Comment by daniel31x13** [comment]
@marcelklehr yes it does but the API isn’t yet properly documented, please DM on discord for a more synchronous chat.

**Comment by daniel31x13** [comment]
**Update:** Floccus integration is now in beta! 🎉 [v5.3.0-beta.1](https://github.com/floccusaddon/floccus/releases/tag/v5.3.0-beta.1)

Huge thanks to @marcelklehr for his work <3

**Comment by marcelklehr** [comment]
final release of v5.3.0 of floccus is out now with linkwarden support. Happy syncing!

**Comment by MAngel666** [comment]
Thats really cool! But do I need to create an extra collection for floccus? It would be nice to sync all linkwarden links and use collections as group. Possible?

**Comment by MAngel666** [comment]
> final release of v5.3.0 of floccus is out now with linkwarden support. Happy syncing!
Thats really cool! But do I need to create an extra collection for floccus? It would be nice to sync all linkwarden links and use collections as group. Possible?

**Comment by MAngel666** [comment]
> final release of v5.3.0 of floccus is out now with linkwarden support. Happy syncing!

Thats really cool! But do I need to create an extra collection for floccus? It would be nice to sync all linkwarden links and use collections as group. Possible?

**Comment by marcelklehr** [comment]
> But do I need to create an extra collection for floccus? 

Yes, you will need to move your collections into a collection that is synced with floccus, because linkwarden does not support top-level bookmarks that are in no collection.

**Comment by MAngel666** [comment]
hmmm and can I use several collections as groups?

**Comment by marcelklehr** [comment]
You can nest collections inside the collection that you sync like normal. Not sure what you mean by groups

**Comment by MAngel666** [comment]
I didn’t see the nested collection option. That’s exactly what I was looking for - PERFECT!
